responsibleai/tests/causal_manager_validator.py

```diff
@@ -57,6 +59,9 @@ def validate_causal(rai_insights, data, target_column,
     assert len(results) == 1
     _check_causal_result(results[0])
 
+    _check_causal_properties(rai_insights.causal.list(),
+                             expected_causal_effects=1)
```

Comment:
Is there a test case for when nothing is supposed to be there?

Reply:
Done. Added the checking of properties when nothing is computed.